### PR TITLE
feature/player

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,6 +3,9 @@ setuptools==56.2.0
 wheel==0.36.2
 twine==3.4.1
 flake8==3.9.2
+pydantic==1.8.1
 pylint==2.8.2
 pytest==6.2.4
+pytest-asyncio==0.15.1
 pytest-cov==2.11.1
+python-jose==3.2.0
```
Comment (iTitus) [inline requirements.txt:11]:
```suggestion
python-jose[cryptography]==3.2.0
```
-> https://pypi.org/project/python-jose/#description

File: whist/core/player.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        :param database: where to look for the user
+        :type database: dictionary
+        :param username: the name of the user to look for
+        :type username: string
+        :return: The player instance or None
+        :rtype: Player or None
+        """
+        if username in database:
+            user: Player = database[username]
+            return Player(**user.dict())
```
Comment (iTitus) [inline whist/core/player.py:26]: why not copy explicitly?

File: whist/core/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"""
+Retrieves envs for the whist core module.
+"""
+import os
+
+ALGORITHM = os.getenv('ALGORITHM', 'HS256')
+SECRET_KEY = os.getenv('SECRET_KEY', 'geheim')
```
Comment (iTitus) [inline whist/core/__init__.py:7]: any reason why to use this algorithm?
just curious

File: tests/whist/core/base_test_case.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+USERNAME = 'honk'
+
+
+class BaseTestCase(unittest.TestCase):
+    def setUp(self) -> None:
+        user = User(user_id=1, username=USERNAME)
+        user_dict: dict = user.dict()
+        user_dict.update({'level': 1})
+        self.player = Player(**user_dict)
```
Comment (iTitus) [inline tests/whist/core/base_test_case.py:14]: why not directly create `Player` ?

**Review by iTitus — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (Segelzwerg) [reply requirements.txt:11]: When I tried this locally it did not work.

File: whist/core/__init__.py (hunk 3 above)
Comment (Segelzwerg) [reply whist/core/__init__.py:7]: Copied from documentation: https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/

File: requirements.txt (hunk 1 above)
Comment (iTitus) [reply requirements.txt:11]: It's defined here: https://github.com/mpdavis/python-jose/blob/master/setup.py#L25
And the package exists: https://pypi.org/project/cryptography/
you might have done something wrong

**Review by iTitus — APPROVED** [review]

**Pull request merged**